src/Provider/GoogleMaps/GoogleMaps.php

```diff
@@ -304,6 +309,20 @@ private function updateAddressComponent(AddressBuilder $builder, string $type, $
                 $builder->addAdminLevel(intval(substr($type, -1)), $values->long_name, $values->short_name);
                 break;
 
+            case 'sublocality_level_1':
+            case 'sublocality_level_2':
+            case 'sublocality_level_3':
+            case 'sublocality_level_4':
+            case 'sublocality_level_5':
+                $newSubLocalityLevel[] = [
+                    'level' => intval(substr($type, -1)),
+                    'name' => $values->long_name,
+                    'code' => $values->short_name,
+                ];
+                $subLocalityLevel = ($builder->hasValue('subLocalityLevel')) ? $builder->getValue('subLocalityLevel', []) : [];
+                $builder->setValue('subLocalityLevel', array_merge($newSubLocalityLevel, $subLocalityLevel));
```

Comment:
Well $subLocalityLevel array is defined only in this function, and doesn't remember it's previous value.. and setter 
``
$builder->setValue('subLocalityLevel',$subLocalityLevel)
``
can set only one value, not an array of values. It override previous value each time and set the new one...so at the end it will contains only last detected subLocalityLevel. Thats why I read previous value, and merge it to new one. Maybe, we should write new function for setting multidimensional array as value?